Examples/RKSearchExample/RKSearchExample.xcodeproj/project.pbxproj

```diff
@@ -338,7 +338,7 @@
 				GCC_WARN_ABOUT_RETURN_TYPE = YES;
 				GCC_WARN_UNINITIALIZED_AUTOS = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 6.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 8.1;
```

Comment:
Per your commit, I noticed that we are trying to support iOS8.0 and above, instead of 8.1, and that is great.  However it looks like this needs to be taken down to 8.0 also.